### PR TITLE
docs: improve UX of the Azure AI Studio Hub setup

### PR DESCRIPTION
**Pull request by aliphys**
I've been reviewing the tutorials (thanks @sdgilley 😀) and made some revisions based on my experience as a (new) user:

1️⃣ The purpose of creating a _Hub_ in Azure AI Studio can be unclear, especially for new users. Furthermore, the distinction from a _project_ may cause confusion (it did for me at first 🙂). To address this, I've added a brief description and a link to further documentation for those interested in 7070d41b1b6ee8e323f7279e19d44e6930e02b42

2️⃣ The "All Hubs" button is not visible in the left panel, which can be confusing for beginners 🤔. I’ve clarified the process for a workflow that worked consistently for me: users should click "All resources & projects" and then "New Hub" to create a Hub. While not the most ergonomic, it is the safest approach IMHO. b1782c485dc414fc0bcde245983bda0fb9f88d4d

3️⃣ The subscription field is empty in the screenshot, which may confuse users, especially since it’s marked as required. I assume the text was manually removed? 94e681132bef05cc4743238004dff28bf8c416cb

4️⃣ The guide assumes users either have no prior resources or are experts. I've added clarifications to account for users with existing resources elsewhere in Azure, specifying the need to enter Hub name, Subscription, Resource group, Location, and Azure AI services. b0b1ae7cbd2c79be456f9df051bbc971eb4dd25f

### CODE REVIEW

**Comment by ShannonLeavitt** [comment]
@aliphys - When your changes are ready to publish, please remove draft mode by selecting **Ready for review**.

image



@sdgilley - Can you review the proposed changes? 

IMPORTANT: When the changes are ready for publication, adding a `#sign-off` comment is the best way to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by aliphys** [comment]
> @aliphys please read the following Contributor License Agreement(CLA). If you agree with the CLA, please reply with the following information.
> 
> ```
> @microsoft-github-policy-service agree [company="{your company}"]
> ```
> 
> > Options:
> > 
> > * (default - no company specified) I have sole ownership of intellectual property rights to my Submissions and I am not making Submissions in the course of work for my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > 
> > * (when company given) I am making Submissions in the course of work for my employer (or my employer has intellectual property rights in my Submissions by contract or applicable law). I have permission from my employer to make Submissions and enter into this Agreement on behalf of my employer. By signing below, the defined term “You” includes me and my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree company="Microsoft"
> > ```
> 
> Contributor License Agreement

@microsoft-github-policy-service agree

File: articles/ai-studio/includes/create-hub.md [hunk 1]
```diff
@@ -10,14 +10,20 @@ ms.date: 5/21/2024
...
 
-1. In the **Create a new hub** dialog, enter a name for your hub (such as *contoso-hub*) and select **Next**. Leave the default **Connect Azure AI Services** option selected. A new AI services connection is created for the hub.
+1. In the **Create a new hub** dialog, enter a name for your hub (such as *contoso-hub*). A new **Resource group** will be created linked to the **Subscription** provided. Leave the default **Connect Azure AI Services** option selected. Select **Next**. A new resource group (*rg-contoso*), and Azure AI service (*ai-contoso-hub*) is created for the hub.
+
+> [!NOTE]
+> If you don't see (new) before the **Resource group** and **Connect Azure AI Services** entries then an existing resource is being used. For the purposes of this tutorial, create a seperate entity via **Create new resource group** and **Create new AI Services**. This will allow you to prevent any unexpected charges by deleting the entities after the tutorial.
 
     :::image type="content" source="../media/how-to/hubs/hub-new-connect-services.png" alt-text="Screenshot of the dialog to connect services while creating a new hub." lightbox="../media/how-to/hubs/hub-new-connect-services.png":::
```
Comment (aliphys) [inline articles/ai-studio/includes/create-hub.md:26]: @sdgilley The linter says that there is an indented code block on line 28. Can you help me resolve it?

File: articles/ai-studio/includes/create-hub.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ---
 title: include file
 description: include file
-author: sdgilley
+author: sdgilley, aliphys
```
Comment (sdgilley) [inline articles/ai-studio/includes/create-hub.md]:
```suggestion
author: sdgilley
```

File: articles/ai-studio/includes/create-hub.md [hunk 3]
```diff
@@ -10,14 +10,20 @@ ms.date: 5/21/2024
...
 To create a hub in [Azure AI Studio](https://ai.azure.com), follow these steps:
 
 1. Go to the **Home** page in [Azure AI Studio](https://ai.azure.com) and sign in with your Azure account.
-1. Select **All hubs** from the left pane and then select **+ New hub**.
+1. Select **All resources** on the left pane. If you cannot see this option, in the top bar select **All resources & projects**. Then select **+ New hub**.
 
     :::image type="content" source="../media/how-to/hubs/hub-new.png" alt-text="Screenshot of the button to create a new hub." lightbox="../media/how-to/hubs/hub-new.png":::
 
-1. In the **Create a new hub** dialog, enter a name for your hub (such as *contoso-hub*) and select **Next**. Leave the default **Connect Azure AI Services** option selected. A new AI services connection is created for the hub.
+1. In the **Create a new hub** dialog, enter a name for your hub (such as *contoso-hub*). A new **Resource group** will be created linked to the **Subscription** provided. Leave the default **Connect Azure AI Services** option selected. Select **Next**. A new resource group (*rg-contoso*), and Azure AI service (*ai-contoso-hub*) is created for the hub.
```
Comment (sdgilley) [inline articles/ai-studio/includes/create-hub.md]:
```suggestion
1. In the **Create a new hub** dialog, enter a name for your hub (such as *contoso-hub*). If you don't have a resource group, a new **Resource group** will be created linked to the **Subscription** provided. Leave the default **Connect Azure AI Services** option selected. 
1. Select **Next**. If you didn't reuse an existing resource group, a new resource group (*rg-contoso*) is created.  Also an Azure AI service (*ai-contoso-hub*) is created for the hub.
```

File: articles/ai-studio/includes/create-hub.md [hunk 4]
```diff
@@ -10,14 +10,20 @@ ms.date: 5/21/2024
...
+
+> [!NOTE]
```
Comment (sdgilley) [inline articles/ai-studio/includes/create-hub.md]: You need to indent this note so it appears under the numbered item.  Then the indented image will no longer be a problem as it is also now under the numbered item.

**Review by sdgilley — COMMENTED** [review]
Thanks for your contributions.  

See my suggestions and comments. 
Also, the Subscription name was blanked out on purpose, don't replace that image.

File: articles/ai-studio/includes/create-hub.md (hunk 4 above)
Comment (aliphys) [reply articles/ai-studio/includes/create-hub.md]: ✅ Indented note in commit [ff7dd61](https://github.com/MicrosoftDocs/azure-ai-docs/pull/55/commits/ff7dd61a172b16272f435f37405c76f9e17fd0d8)

**Comment by aliphys** [comment]
Thanks for the feedback @sdgilley 🙂
#sign-off

**Comment by Jak-MS** [comment]
@aliphys - if you're done making changes, let @sdgilley know so she can review/approve and sign off if it's ready. thanks

**Comment by aliphys** [comment]
I'm done. 😊

**Comment by aliphys** [comment]
@sdgilley could you re-review this PR at a suitable time? I've addressed your comments. I would use the #sign-off tag, but I get an error (see comment below by bot). Thank you! 🙂

**Comment by ShannonLeavitt** [comment]
@aliphys - Can you please also remove the yellow highlighting from *articles/ai-studio/media/how-to/hubs/hub-new-connect-services.png*?

@sdgilley - After @aliphys removes the yellow highlight in the image, the commits will need to be pulled into the private repo for staging to ensure that the images render properly and for automated checks.

**Comment by aliphys** [comment]
@ShannonLeavitt highlight over the `Create new AI Services` removed in commit [3396659](https://github.com/MicrosoftDocs/azure-ai-docs/pull/55/commits/339665981502de17124ea2681d053821d72dcfd3) ✅

**Comment by sdgilley** [comment]
I've pulled these changes into https://github.com/MicrosoftDocs/azure-ai-docs-pr/pull/1270.

**Comment by v-dirichards** [comment]
> I've pulled these changes into [MicrosoftDocs/azure-ai-docs-pr#1270](https://github.com/MicrosoftDocs/azure-ai-docs-pr/pull/1270).

This PR is now merged.

**Comment by sdgilley** [comment]
@aliphys  - your changes will be live later today.  Thanks for your patience in getting this merged.  
#please-close

**Pull request closed without merging**